Source/Resources/Content/MSGraph/DeviceManagement/DeviceManagementScripts/Management - Set-IntuneManagementExtensionConfiguration.json

```diff
@@ -7,7 +7,7 @@
       }
     }
   ],
-  "description": "Configures the IntuneManagementExtension.\n\nDisables the recycleScripts setting so that the Content and Policies directories under C:\\Program Files (x86)\\Microsoft Intune Management Extension do not get automatically cleaned out by the extension.",
+  "description": "What is it: Configures the IntuneManagementExtension. Disables the recycleScripts setting so that the Content and Policies directories under 'C:\\Program Files (x86)\\Microsoft Intune Management Extension' do not get automatically cleaned out by the extension. \n Simeon Notes: This assists in troubleshooting any device enrollment errors.",
```

Comment:
**What does this do?**
The baseline configures the IntuneManagementExtension to not delete installation logs from devices. 

**Why should you use this?**
If you want to have logs available to troubleshoot device management issues.

**What is the end-user impact?**
Users will be able to view installation logs.

**Learn more**
NA